trunk/SUAVE/Methods/Weights/Correlations/BWB/empty.py

```diff
@@ -0,0 +1,206 @@
+# empty.py
+# 
+# Created:  Apr 2017, M. Clarke 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units, Data
+from cabin import cabin
+from aft_centerbody import aft_centerbody
+from landing_gear import landing_gear
+from payload import payload
+from systems import systems
+from wing_main import wing_main
+from SUAVE.Methods.Weights.Correlations import Propulsion as Propulsion
+import warnings
+
+# ----------------------------------------------------------------------
+#  Empty
+# ----------------------------------------------------------------------
+
+def empty(vehicle):
+    """ output = SUAVE.Methods.Weights.Correlations.Tube_Wing.empty(engine,wing,aircraft,fuselage,horizontal,vertical)
```

Comment:
I updated the script but forgot to change the comments. This issue has been rectified